.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run husky_msg && npx lint-staged && npm run compile_css
+npm run husky_msg && npx lint-staged
```

Comment:
I moved the `compile_css` step into the lint-staged actions so it only runs when required